Cargo.toml

```diff
@@ -18,6 +18,8 @@ exclude = [
 [features]
 # Use a spinlock internally (may be faster on some platforms)
 spin = []
+# Use a spinlock internally without `std::thread::sleep` (for platforms that don't support blocking)
```

Comment:
I'm unconvinced that this should be a feature. I think it should just be possible to switch the use of `thread::sleep` based on whether the target is one we support or not.

Reply:
Alright, I removed the feature and changed it so that the use of `thread::sleep` in the spinlock implementation is only enabled if `cfg!(any(target_family = "unix", target_family = "windows"))` is true